blog/content/post/2025-05-05-aem_uefi.md

```diff
@@ -0,0 +1,232 @@
+---
+title: TrenchBoot AEM gains support for UEFI installations
+abstract: 'A feature craved by many is finally here.
+          AEM can now be used with Qubes OS installed under UEFI.
+          Oh, and some automated testing. But mostly UEFI.'
+cover: /covers/trenchboot-logo.png
+author: krystian.hebel
+layout: post
+published: true
+date: 2025-05-05
+archives: "2025"
+
+tags:
+  - QubesOS
+  - TrenchBoot
+  - GRUB2
+  - Xen
+  - Testing
+categories:
+  - Firmware
+  - Bootloader
+  - Hypervisor
+  - OS Dev
+  - Security
+
+---
+
+## What's new in TrenchBoot AEM?
+
+Adding support for UEFI installations was the main goal of this part of the
+project [funded by NLnet foundation](https://nlnet.nl/project/TrenchBoot-AEM-UEFI/).
+An approach similar to that of Linux (which is currently [being upstreamed](https://lore.kernel.org/lkml/20250421162712.77452-1-ross.philipson@oracle.com/))
+was used. Even though this isn't how Xen is normally booted with Qubes OS, it
```

Comment:
```suggestion
was used (the details are below). Even though this isn't how Xen is normally booted with Qubes OS, it
```

Reply:
https://github.com/3mdeb/news-and-ideas/pull/660/commits/e5f849eb640d87305e2af15f61dbec065e0363f2